tests/e2e/login/privacy-policy.wdio-spec.js

```diff
@@ -0,0 +1,128 @@
+
+      // After accepting, no privacy policy on next load
+      await privacyPolicyPage.acceptPrivacyPolicy();
+      await browser.url('/');
+      expect(await privacyPolicyPage.privacyWrapper()).not.toBeDisplayed();
+
+      // Check display when loading privacy policy page
+      await privacyPolicyPage.goToPrivacyPolicyConfig();
+      await browser.waitUntil(async () => {
+        const wrapperText = await (await privacyPolicyPage.privacyConfig()).getText();
+        return wrapperText.includes(englishTexts.header) && wrapperText.includes(englishTexts.paragraph);
+      }, 'Timed out waiting for english online privacy to display');
+
+      // No privacy policy on 2nd login
+      await browser.reloadSession();
+      await browser.url('/');
+      await loginPage.login(onlineUser.username, onlineUser.password);
+      expect(await privacyPolicyPage.privacyWrapper()).not.toBeDisplayed();
+
+
+      // After login in french, check that privacy policy was prompted to user again
+      await browser.reloadSession();
+      await browser.url('/');
+      await loginPage.login(onlineUser.username, onlineUser.password, 'fr');
+      await browser.waitUntil(async () => {
+        const wrapperText = await (await privacyPolicyPage.privacyWrapper()).getText();
+        return wrapperText.includes(frenchTexts.header) && wrapperText.includes(frenchTexts.paragraph);
+      }, 'Timed out waiting for french online privacy to display');
+      await privacyPolicyPage.acceptPrivacyPolicy();
+    });
+  });
+
+
+  // WDIO currently cannot log in offline users because of service worker race condition
+  // https://github.com/medic/cht-core/issues/7242
+  xdescribe('for a french offline user', () => {
+    let passed = false;
+    afterEach(async () => {
+      if (!passed) {
+        // I suspect this test is failing because of a conflict.
+        const userDoc = await utils.requestOnTestDb('/org.couchdb.user:offline?conflicts=true');
+        console.log('Check if the test failed because of a conflict on this doc:');
+        console.log(JSON.stringify(userDoc, null, 2));
+      }
+    });
+
+    xit('should show the correct privacy policy on login', async () => {
+      // After first login in french, check that privacy policy was prompted to user
+      await browser.reloadSession();
+      await browser.url('/');
+      await loginPage.login(offlineUser.username, offlineUser.password, 'fr');
+      await browser.waitUntil(async () => {
+        const wrapperText = await (await privacyPolicyPage.privacyWrapper()).getText();
+        return wrapperText.includes(frenchTexts.header) && wrapperText.includes(frenchTexts.paragraph);
+      }, 'Timed out waiting for french offline privacy to display');
+
+      // After accepting, no privacy policy on next load
+      await privacyPolicyPage.acceptPrivacyPolicy();
+      await commonElements.sync();
+
+      await browser.reloadSession();
```

Comment:
Did you mean: 
```suggestion
      await browser.refresh();
```
`reloadSession` creates a completely new session (no login no nothing).. Ofc we don't see the privacy policy.